cdap-security/src/test/java/co/cask/cdap/security/authorization/InMemoryAuthorizer.java

```diff
@@ -168,24 +170,32 @@ public void removeRoleFromPrincipal(Role role, Principal principal) throws RoleN
         privileges.addAll(getPrivileges(role));
```

Comment:
Also, will `getPrivileges(role)` works? When adding a role, I don't see how it affect the `privileges` map, which is the only map that the `getPrivileges` method uses.